libs/vue-route-guard/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "strict": true,
+    "allowJs": true,
```

Comment:
why `allowJs` is `true`?

Reply:
i will remove it, it was part of the default setting configured by the package i used to setup the project